code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -794,13 +785,15 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	if(issilicon(usr))
 		return
 
-	if(can_use(usr))
+	if(usr.canUseTopic(src))
 		if(id)
 			remove_id()
 		else
 			usr << "<span class='warning'>This PDA does not have an ID in it!</span>"
-	else
+	if (usr.restrained())
 		usr << "<span class='warning'>You cannot do that while restrained!</span>"

```

Comment:
yeah it's okay don't worry, we have other ways to tell the player that they are restrained or stunned or whatever